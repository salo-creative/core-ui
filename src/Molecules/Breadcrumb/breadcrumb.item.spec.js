import React from 'react';
import 'jest-styled-components';

import { renderWithTheme } from '../../../test';

// COMPONENT
import BreadcrumbItem from './breadcrumb.item';

test('Renders against snapshot', async () => {
  const { container } = renderWithTheme(
    <BreadcrumbItem />
  );
  expect(container.firstChild).toMatchSnapshot();
});