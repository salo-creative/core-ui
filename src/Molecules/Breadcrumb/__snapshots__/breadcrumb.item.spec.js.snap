// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders against snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 100%;
  font-size: 1.4rem;
}

.c0 a {
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #006287;
  -webkit-transition: color 0.3s linear;
  transition: color 0.3s linear;
}

.c0 a:hover {
  color: #00ACED;
}

.c1 {
  color: #6c757d;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0 1rem;
  height: 100%;
}

<span
  class="c0"
>
  <span
    class="c1"
  >
    /
  </span>
</span>
`;
